### ISSUE TITLE
reCAPTCHA v3 support

### ISSUE THREAD

**Issue by kukac7** [DethAriel/ng-recaptcha#129]
> reCAPTCHA v3 returns a score for each request without user friction. The score is based on interactions with your site and enables you to take an appropriate action for your site.

https://developers.google.com/recaptcha/docs/v3

**Comment by vikingschris** [comment]
Does this library support v3? If so, do you have an example how to do so with this library?

**Comment by DethAriel** [comment]
@kukac7 thx for submitting this, and sorry for taking so long to respond.

I'll be working on adding v3 support in the next couple of days, and you should see it in the next minor release of `ng-recaptcha`, which will be `4.3.0`

**Comment by DethAriel** [comment]
@kukac7 @vikingschris and all,

I've released reCAPTCHA v3 support in `4.3.0-beta.1`, and you can use it starting today! It's still in the beta stage because there are some more changes that I would like to get into final `4.3.0` (like deprecations of some APIs), but the public API for reCAPTCHA v3 service is unlikely to change at this point (unless there are outstanding issues found by early adopters).

The documentation can be found in the "vnext" branch (see [here](https://github.com/DethAriel/ng-recaptcha/tree/vnext#example-basic-v3)). I will update the demo site when I publish the actual 4.3.0.

If you have any feedback on the API, or any bug reports - feel free to add a comment to this thread, and I'll do my best to address it ;)

Happy coding, everybody!

**Comment by stuurman** [comment]
Thanks for your great work!
already any idea when 4.3.0 is scheduled to be released?

**Comment by mskorb** [comment]
any update for 4.3.0 release?

**Comment by LucasBrazi06** [comment]
I used the V3 using this code:

```
import {ReCaptchaV3Service} from 'ng-recaptcha';

constructor(
    private reCaptchaV3Service: ReCaptchaV3Service) {
}

resetPassword(data) {
  this.reCaptchaV3Service.execute('Reset').subscribe((token) => {
    ... 
  });
|
```

And I get this error on build and deployed in prod env:

![image](https://user-images.githubusercontent.com/25266945/58754342-27f4e100-84ce-11e9-89ef-58c145f23444.png)

Any ideas?
Thanks in advance.

**Comment by DethAriel** [comment]
@mskorb just released 4.3.0, so feel free to update!

**Comment by DethAriel** [comment]
@LucasBrazi06 the same code is used in the examples site (see [here](https://dethariel.github.io/ng-recaptcha/v3)), and it seems that it works fine. What makes you think that `ng-recaptcha` is the source of the error that you're seeing?
